### DISCUSSION TITLE
C57E9FF1-07E5-4AC4-B50E-0782FF1BA91A

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#47] [General]
# C57E9FF1-07E5-4AC4-B50E-0782FF1BA91A

The study that confirmed TOI-1136 also mentioned a seventh planet candidate (.07= 2.5 Re), though they mention that the existence of the planet had little impact on the data of TOI-1136, and thus wasn't included in the final analysis, they also added that they only found the planet after a visual overview of the data and the BLS analysis didn't find it.
If it does ever get confirmed, then TOI-1136 would tie with TRAPPIST-1 for the second most amount of planets orbiting a star.

NOTE: In order for this addon to work properly, an existing TOI-1136 addon has to be installed.

https://celestia.mobi/addon?item=C57E9FF1-07E5-4AC4-B50E-0782FF1BA91A&lang=en

**Comment by exocritic5457** [comment]
One of these days another system will rise up against TRAPPIST-1... And frankly, if that ever does manage to happen, I can't wait

**Comment by exocritic5457** [comment]
the year is ceasing, and this candidate and its single transit still haven't bit the dust. I'm still hopeful for this system after Kepler-385 happened
